browser/importer/brave_profile_writer.cc

```diff
@@ -36,3 +40,14 @@ void BraveProfileWriter::AddCookies(
         network::mojom::CookieManager::SetCanonicalCookieCallback());
   }
 }
+
+void BraveProfileWriter::UpdateStats(const BraveStats& stats) {
+  PrefService* prefs = profile_->GetOriginalProfile()->GetPrefs();
+
+  prefs->SetUint64(kAdsBlocked,
```

Comment:
I'm not going to block on this @garrettr but if you have time for a follow up I'd prefer to have this take the greater of the 2 stats and use that. That way someone can't as easily game it by importing multiple times.

Reply:
@bbondy Blocking on that is no problem, it's a quick fix! I'll do it right now

Reply:
I was too fast for you, so you'll have to PR again, but can be from the same branch obviously.

Reply:
actually I wasn't too fast for you but I was too slow to read your reply and too fast to merge 😆